README.md

```diff
@@ -30,7 +30,7 @@ with fixes will be most welcome!
 You will need to register / request a tunnel from either tunnelbroker.net or
 6project.org.
 
-> **Disclaimer**  
+> **Disclaimer**
```

Comment:
The double space at the end of a line means a line break in markdown. Before this change, it would render as:

> **Disclaimer**
> _This project is not associated ..._

After this change, it renders as:

> **Disclaimer** _This project is not associated ..._

Did you actually mean to remove the line break?